libs/blocks/quiz/utils.js

```diff
@@ -5,6 +5,7 @@ import { getConfig } from '../../utils/utils.js';
 const QUESTIONS_EP_NAME = 'questions.json';
 const STRINGS_EP_NAME = 'strings.json';
 const RESULTS_EP_NAME = 'results.json';
+const VALID_URL_RE = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/;
```

Comment:
We should either expect this to be true from authors or have checks in more meaningful places like before publish.

Reply:
Hi @JackySun9. I would ditch the prefixHttps method. As @fullcolorcoder suggests, just expect a valid URL. To that end, if you update the regex as follows, I believe it will enforce https and lowercase characters. 
`
/^(http(s):\/\/.)[-a-z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-z0-9@:%_+.~#?&//=]*)/
`